website/src/content/docs/pt-br/internals/philosophy.mdx

```diff
@@ -1,42 +1,42 @@
 ---
-title: Philosophy
-description: How we think about building Biome.
+title: Filosofia
+description: Como pensamos em construir o Biome
 ---
 
-This list includes general ethos that the project should abide by.
-This list is not comprehensive. Some of these are obvious but are stated for completeness.
+Esta lista inclui princípios gerais que o projeto deveria respeitar.
+A lista não é abrangente. Alguns desses princípios são óbvios, mas são mencionados para garantir a completude.
 
-## Project Management
+## Gestão do Projeto
 
-- **Set clear expectations.** Make project intent and decisions known well in advance.
-  Nothing should be a surprise.
+- **Estabelecer expectativas claras.** Torne a intenção do projeto e suas decisões conhecidas com bastante antecedência.
```

Comment:
```suggestion
- **Estabelecer expectativas claras.** Mostrar as intenções  e decisões do projeto com bastante antecedência.
```

Small suggestion, what do you think?

Reply:
Looks good to me. Applied on [4c15c52](https://github.com/biomejs/biome/pull/1139/commits/4c15c52c9e9a0dbf6c7e71ae777110941d5903dd)